doc/examples/filters/plot_enhancement_measure.py

```diff
@@ -0,0 +1,61 @@
+"""
+=========================================
+Quantitative Measure of Image Enhancement
+=========================================
+In this example, we will quantify the effect of applying contrast enhancement
+to an image using a measure called EME [1]_.
+
+It is defined as the log of the ratio of local maximum to minimum within a
+sliding window of given size. This measure aims to provide a perceptual
+estimate of image quality, useful when comparing image enhancement algorithms.
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage import data
+from skimage.filters import gaussian
+from skimage.exposure import equalize_adapthist
+from skimage.metrics import simple_metrics
+
+img_gray = data.camera()
+img_rgb = data.astronaut()
+
+
+def compare_side_by_side(before: np.ndarray, after: np.ndarray):
+    fig, ax = plt.subplots(nrows=1, ncols=2, figsize=(16, 14))
+    for idx, image in enumerate([before, after]):
+        ax[idx].imshow(image)
+        ax[idx].axis('off')
+        ax[idx].set_title("EME = {:.2f} ".format(
+            simple_metrics.enhancement_measure(image, size=3)))
+    fig.show()
+
+
+#####################################################################
+# We will create a function that takes an original image as `before`
+# and applies a `transform` to it. We will use a gaussian filter to
```

Comment:
Nitpick: the function doesn't actually apply a transform to it. It just calls `enhancement_measure` on both and plots the results. Rewording this might be a lot clearer to new users.